packages/neuron-ui/src/components/PricePanel/index.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { localNumberFormatter, shannonToCKBFormatter } from 'utils'
+import RingProgressBar from 'widgets/RingProgressBar'
+import TextField from 'widgets/TextField'
+import { Transfer } from 'widgets/Icons/icon'
+import DropdownWithCustomRender from 'widgets/DropdownWithCustomRender'
+import { useTranslation } from 'react-i18next'
+import { appState, useState as useGlobalState } from 'states'
+
+import { useGetbatchGeneratedTx } from 'components/Send/hooks'
+import { DEFAULT_COUNT_DOWN } from 'utils/const'
+
+import styles from './pricePanel.module.scss'
+
+interface PricePanelProps {
+  field: string
+  price: string
+  onPriceChange: React.EventHandler<React.SyntheticEvent>
+}
+enum PriceTypeEnum {
+  Customer = 'customer',
+  Standard = 'standard',
+}
+const DEFAULT_PRICE_ARRAY = ['1000', '2000', '3000']
+
+const PricePanel: React.FunctionComponent<PricePanelProps> = ({ price, field, onPriceChange }: PricePanelProps) => {
+  const [t] = useTranslation()
+  const [type, setType] = useState<PriceTypeEnum>(PriceTypeEnum.Standard)
+  const [inputError, setInputError] = useState<string | null>(null)
+  const [inputHint, setInputHint] = useState<string | null>(null)
+  const [dropdownValue, setDropdowValue] = useState(DEFAULT_PRICE_ARRAY[1])
+  const [feeMap, setFeeMap] = useState([])
+  const [priceArray, setPriceArray] = useState(DEFAULT_PRICE_ARRAY)
+  const {
+    app: {
+      send = appState.send,
+      feeRateStatics: { suggestFeeRate = 0 },
+      countDown,
+    },
+    wallet: { id: walletID = '' },
+  } = useGlobalState()
+
+  const isStandard = type === PriceTypeEnum.Standard
+  const label = isStandard ? '单价' : '自定义价格'
+  const percent = (countDown / DEFAULT_COUNT_DOWN) * 100
+
+  const handleDropdownChange = useCallback((e: any) => {
+    const { value: dropdownChangedValue } = e
+    setDropdowValue(dropdownChangedValue)
+    onPriceChange?.({ target: { value: dropdownChangedValue } } as any)
+  }, [])
+
+  const handleInputChange = useCallback((e: any) => {
+    const { value: inputChangedValue } = e.target
+    const handledValue = inputChangedValue.replace(/,/g, '')
+
+    setInputError(Number(handledValue) < 1000 ? '价格设定最少不得少于 1000 shannons/byte，请重新输入' : null)
+    onPriceChange?.(e)
+  }, [])
+
+  const feeValuesMap = feeMap?.map((item: any) => ({
+    feeRateValue: item.feeRateValue,
+    value: shannonToCKBFormatter(item.feeValue),
+  }))
+
+  useEffect(() => {
+    if (!priceArray.includes(dropdownValue)) {
+      handleDropdownChange({ value: priceArray[1] })
+    }
+  }, [priceArray])
```

Comment:
Acctually, it will be updated when the suggestFeeRate is updated.

Reply:
It's strange that when the `suggestFeeRate` changes, it will change the selected option.
I think if the user selects `fast`, it will also be `fast`. 
Or, to put it another way, the `suggestFeeRate` only affects the display, but should not affect the selected option.
And in current logic, it may exist one exception, after the user selects the fast, and the current fee is 3000, but later, the fast fee become 2000, it will change to the slow option, it's strange, the user selects fast, but it becomes slow.

Reply:
Ok, have fixed the FeeRateMode which is currently selected.